packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html

```diff
@@ -264,8 +247,12 @@ <h3 data-translate="telecom_pack_migration_doing"></h3>
 
             <p
                 data-ng-if="$ctrl.process.selectedOffer.needNewModem && $ctrl.process.shipping.mode === 'transporter'"
-                data-translate="telecom_pack_migration_confirm_shipping"
-                data-translate-values="{ price : $ctrl.getFirstMensuality().text, transportPrice : $ctrl.modemTransportPrice }"
+                data-ng-bind-html="$ctrl.displayTranslatedPrice('telecom_pack_migration_confirm_shipping', $ctrl.modemTransportPrice)"
+            ></p>
+
+            <p
+                data-ng-bind-html="$ctrl.displayTranslatedPrice('telecom_pack_migration_confirm_first_mensuality', $ctrl.process.selectedOffer.firstMensuality.text)"
+                class="font-weight-bold"
```

Comment:
**nitpick:** `translate` directive can be used directly used here.

Reply:
I made a function because with the `class` argument in the span, the data-translate don't work.